catalog/ftp/src/main/java/ddf/catalog/ftp/UserManagerImpl.java

```diff
@@ -98,9 +103,15 @@ public User authenticate(Authentication authentication) throws AuthenticationFai
     if (authentication instanceof UsernamePasswordAuthentication) {
       username = ((UsernamePasswordAuthentication) authentication).getUsername();
```

Comment:
✏️ Just a little light nit - We could just save this off to a variable instead of recasting it 3 times...